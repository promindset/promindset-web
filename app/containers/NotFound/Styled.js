import styled from 'styled-components'

export const Styled = styled.div`
  margin-top: 10vh;
  min-height: 50vh;
  h1 {
    text-align: center;
  }
`
