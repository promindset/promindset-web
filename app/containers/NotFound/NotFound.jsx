import React from 'react'
import { Styled } from './Styled'

export const NotFound = () => (
  <Styled>
    <h1>Page Not Found</h1>
  </Styled>
)
