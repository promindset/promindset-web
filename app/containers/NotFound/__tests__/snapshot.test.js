import React from 'react'
import renderer from 'react-test-renderer'
import { NotFound } from '../NotFound'

describe('Not Found Page', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<NotFound />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
