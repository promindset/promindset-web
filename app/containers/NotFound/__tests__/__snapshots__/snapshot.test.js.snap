// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Page snapshot renders 1`] = `
<div
  className="sc-bdVaJa iShcTr"
>
  <h1>
    Page Not Found
  </h1>
</div>
`;
